u3-views/features/videoController.js

```diff
@@ -0,0 +1,18 @@
+const { getVideo, getSuggestions } = require('../_services/fakeapi')
+
+const Video = (req, res) => {
+  const videoId = req.params.id
+  const video = getVideo(videoId)
+  const teases = getSuggestions(6)
+  res.render('video-player', { video: video, teases: teases })
+}
+const Videohome = (req, res) => {
+  const homes = getSuggestions(30)
+
+  res.render('video-home', { homes: homes })
+}
```

Comment:
ดี! ฉันดีใจที่คุณใช้ชื่อตัวแปรอื่นสำหรับ `homes` มันทำให้เห็นชัดเจนขึ้นว่ามีการใช้ในทางที่ต่างออกไป
Good! I'm glad that you used a different variable name for `homes`. It makes it clearer that it's used in a different way.